CHANGES.txt

```diff
@@ -2,6 +2,155 @@
 Chaco CHANGELOG
 ===============
 
+Release 5.0.0
+-------------
+
```

Comment:
let's add the contributors list here - given that we already put it together

Reply:
I've also added all the changes / contributors to https://github.com/enthought/chaco/releases/tag/5.0.0
Should I use full names or github user names for everyone?  Or a mix of both?